libcidr-shopify.rb

```diff
@@ -0,0 +1,19 @@
+class LibcidrShopify < Formula
+  desc 'Libcidr is a library that provides utility functions to handle IP addresses and net blocks.'
+  homepage 'https://github.com/Shopify/libcidr-shopify/'
+  url 'http://bazaar.launchpad.net/~fullermd/libcidr/trunk/tarball/302'
```

Comment:
is this the equivalent of a master branch?
do we want the "stable" release tarball instead from https://www.over-yonder.net/~fullermd/projects/libcidr/libcidr-1.2.3.tar.xz ?

Reply:
@sbfaulkner @ElvinEfendi  @andrewlouis93 There is only a development branch on the version control of the project, with tagged releases: https://code.launchpad.net/libcidr . I decided to go with the dev branch on a specific tagged release because the stable release tarball is hosted by the maintainer (if his hosting goes down, so does the tarball), as opposed to where the tarball I'm currently using is hosted from: ``` http://bazaar.launchpad.net ```. What are your thoughts regarding this?

Reply:
that ui sucks, so I can't (easily) figure out which "release" or commit corresponds to v1.2.3 ... do you have any idea how that works?

Reply:
@sbfaulkner I agree :). Here you can see all the revisions of the project: http://bazaar.launchpad.net/~fullermd/libcidr/trunk/changes.

Reply:
That `Development` branch seems to be the equivalent of the master branch, from looking at the repository. Since it's pretty much the source, I think I'm okay with the package being pulled from bazaar.launchpad.net.

Reply:
for clarity ... we should be linking to the equivalent of the stable 1.2.3 release - is that what 320 is? (I don't even see it in /trunk/changes 😕

Reply:
doh... 302 ... my eyes playing tricks on me

Reply:
and NOW I see the TAG... :-)

Reply:
so, this *is* cool